ui/src/criteria/attribute.tsx

```diff
@@ -1,11 +1,17 @@
-import { Checkbox, FormControlLabel, ListItem } from "@mui/material";
+import { Checkbox, FormControlLabel, ListItem, Slider } from "@mui/material";
+import Box from "@mui/material/Box";
+import Grid from "@mui/material/Grid";
+import Input from "@mui/material/Input";
 import Stack from "@mui/material/Stack";
 import Typography from "@mui/material/Typography";
-import { CriteriaConfig, CriteriaPlugin, registerCriteriaPlugin } from "cohort";
+import { CriteriaPlugin, registerCriteriaPlugin } from "cohort";
 import produce from "immer";
 import React from "react";
 import * as tanagra from "tanagra-api";
+import { CriteriaConfig } from "underlaysSlice";
 import { useUnderlay } from "../hooks";
+import { Underlay } from "../underlaysSlice";
+import { isValid } from "../util/valid";
```

Comment:
Existing issue on L12 too, but please avoid parent-relative imports like this. Relative imports should only be used on imports from the same directory (./foo), otherwise they should just be relative to the application root per L11